modules/nextflow/src/main/groovy/nextflow/dag/DAG.groovy

```diff
@@ -417,6 +417,10 @@ class DAG {
             indexOf(this)
         }
 
+        int getId() {
+            getOrder()
+        }
+
```

Comment:
Maybe `getOrder` can be used. `getId` doesn't look really needed

Reply:
Removed it! Feel free to merge it directly. 
However, introducing an artificial ID independent of the order could avoid potential problems. Should I go for it?

Reply:
Do you mean using an atomic long in the same it has been done for the `Edge`. It could make sense. Please add a couple of small unit tests in that case

Reply:
Exactly. Should I replace the order or have the id additionally?

Reply:
I'd to keep `getOrder` in the current form and add `getId` as you are suggesting

Reply:
Okay, done.